test/modules/acbs/acbs-authentication-failed.exception.test.ts

```diff
@@ -0,0 +1,25 @@
+import { AcbsAuthenticationFailedException } from "../../../src/modules/acbs/acbs-authentication-failed.exception";
```

Comment:
For unit tests, I'd be happy to have the file next to the actual module. 
And have only api-tests in `test` folder.

Reply:
I typically keep unit-tests in `test` folder but don't have an objection to keeping them next to the actual module. I've moved the unit tests and updated the jest config to look for them in the `src` folder

Reply:
Thank you. 
Yes, it will be consistent and it will be easier to separate the testing scopes